requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+git://github.com/fredj/cssmin.git@master#egg=cssmin
+cssmin==0.2.0
```

Comment:
This is an unrelated change and points to a totally different version of cssmin.

Reply:
Can revert this one, but `pip install -r requirements.txt` wouldn't run with it. Couldn't be bothered investigating why so matched the name against pypi and went with it :)